atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/TimestampStateStoreTest.java

```diff
@@ -33,6 +33,18 @@
     private static final UUID LEADER = UUID.randomUUID();
     private static final LockWatchVersion VERSION_1 = LockWatchVersion.of(LEADER, 1L);
     private static final LockWatchVersion VERSION_2 = LockWatchVersion.of(LEADER, 17L);
+    private static final LockToken WRITES_TOKEN = LockToken.of(UUID.randomUUID());
+    private static final LockToken WRITES_TOKEN_2 = LockToken.of(UUID.randomUUID());
+    private static final TransactionUpdate UPDATE = TransactionUpdate.builder()
+            .startTs(100L)
+            .commitTs(400L)
+            .writesToken(WRITES_TOKEN)
+            .build();
+    private static final TransactionUpdate UPDATE_2 = TransactionUpdate.builder()
+            .startTs(102L)
+            .commitTs(402L)
```

Comment:
Is it worth extracting a static at least for these start timestamps, seeing as they _should_ match what is in the map?